app/models/game.rb

```diff
@@ -72,7 +74,7 @@ def self.approved
   end
 
   # Usage example: @game.average_score_for :fun
-  def average_score_for( attribute )
-    reviews.average( attribute ).round(2) * 20
+  def average_score_for(attribute)
+    reviews.average(attribute).round(2) * 20
```

Comment:
Oh, ok. Makes sense. There could be cases where it would be used for other purposes than display 👍